demographics.html

```diff
@@ -112,55 +117,102 @@ <h1>Demographics</h1>
             <!--Charts-->
             <div class="row">
                 <div class="col-md-6 col-sm-6 col-xs-12">
-                    <h4 align="center"> Housing Tenure </h4>
+                    <h4 align="center"> Quick Stats </h4>
+                    <h3>Service Zone <span id="zone-selection">C1</span></h3>
                     <div id="canvas-holder" align="center">
-                        <canvas id="canvas3" width="400" height="300" />
+                        <table class="table table-striped">
```

Comment:
again, code overwriting old code. i think this is because you are responding to a commit-review?

perhaps we need to close these commits and start a new pull request? this looks like you are overwriting the same pull request... maybe that's ok. not sure yet